test/parallel/test-https-connect-address-family.js

```diff
@@ -36,8 +36,13 @@ function runTest() {
 }
 
 dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
-  if (err)
+  if (err) {
+    if (err.code === 'ENOTFOUND') {
+      common.skip('localhost does not resolve to ::1');
```

Comment:
@thefourtheye That's true with or without the change here, right? Like, without the change here, the test fails (but it is marked as passing if run with `test.py` or `make test` because of the setting in `parallel.status`). With the change here, the test is skipped. In both cases, you need to change `/etc/hosts` to get the test to pass (running from the command line without the test runner). Or am I mistaken? If I'm not mistaken, then I think this is an improvement. (Skipping is better than wrongly failing.) 

@bnoordhuis I think we *might* be able to go back to trying to resolve addresses in `common.localIPv6Hosts` once this fix is in there, but I'm not 100% sure. Either way, I'd rather see this (hopefully clear improvement) land, and only then experiment with adding that functionality back in to see if it works any better once this safeguard is there.